collect_app/src/main/java/org/odk/collect/android/instancemanagement/send/ReadyToSendBanner.kt

```diff
@@ -0,0 +1,45 @@
+package org.odk.collect.android.instancemanagement.send
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.lifecycle.LifecycleOwner
+import org.odk.collect.android.databinding.ReadyToSendBannerBinding
+import org.odk.collect.strings.R
+
+const val ONE_SECOND = 1000L
```

Comment:
Maybe makes sense to just pull these out to an object in `shared` seeing as they're generally pretty useful. Could just be called `TimeInMs` or something.

Reply:
Done.